.travis.yml

```diff
@@ -29,3 +32,5 @@ after_success:
   # gover collects all .coverprofile files and saves it to one file gover.coverprofile
   - gover
   - goveralls -coverprofile=gover.coverprofile -service=travis-ci
+  # sync the docs only if everything else was successful
+  - cd script && ./sync-docs.sh
```

Comment:
I guess that this  is because of path `DOCS_KEY`.

I agree with @containscafeine it is not nice to do cd ;-)
How about changing `DOCS_KEY` to `script/deploy_key` and run it like `script/sync-docs.sh`?